Documentation.md

```diff
@@ -0,0 +1,12 @@
+## Endpoints:
+
```

Comment:
its great that you put this in, will definitely help next person, bonus points for caring